evm/chains/base.py

```diff
@@ -589,10 +582,7 @@ def validate_block(self, block: BaseBlock) -> None:
         Since block validation (specifically the uncle validation must have
         access to the ancestor blocks, this validation must occur at the Chain
         level.
-
-        TODO: move the `seal` validation down into the vm.
         """
-        self.validate_seal(block.header)
```

Comment:
Maybe do a `vm.validate_seal()` here instead of trying to merge it into the uncle check?

Reply:
Done